pkg/apis/operator/v1alpha1/catapult_types.go

```diff
@@ -36,8 +36,23 @@ type CatapultSpec struct {
 	// ServiceCluster: Webhook will call webhooks of the CRD in the ServiceCluster with dry-run flag.
 	// +kubebuilder:default:=None
 	WebhookStrategy corev1alpha1.WebhookStrategyType `json:"webhookStrategy,omitempty"`
+	// Paused tell controller to pause reconciliation process and assume that Catapult is ready
+	Paused PausedFlagType `json:"paused,omitempty"`
 }
 
+// PausedFlagType represents a enable/disable flag
+type PausedFlagType string
+
+func (o PausedFlagType) IsPaused() bool {
+	return o == PausedFlagTrue
+}
+
+// Values of PausedFlagType.
+const (
+	PausedFlagTrue  PausedFlagType = "true"
+	PausedFlagFalse PausedFlagType = "false"
```

Comment:
Can you please make these two string values to `True` and `False` Here is the kubernetes api-conventions:https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants

Reply:
Done